cocos/base/ccUtils.cpp

```diff
@@ -326,6 +326,11 @@ Sprite* createSpriteFromBase64Cached(const char* base64String, const char* key)
         CCASSERT(imageResult, "Failed to create image from base64!");
         free(decoded);
 
+        if (!imageResult) {
+            CC_SAFE_RELEASE_NULL(image);
+            return nullptr;
+        }
+
```

Comment:
Yes. `imageResult` variables are already used in a `CCASSERT()` at line 326 and 345 but when compiling libcocos2d in a release build, [`CCASSERT` macros are disabled](https://github.com/cocos2d/cocos2d-x/blob/49e24eb0ba55d1794d49609cb53555cb7fb2977b/cocos/base/ccMacros.h#L52-L54), so these codes produce a warning about unused variables.